app/assets/javascripts/reducers/users.js

```diff
@@ -23,12 +24,32 @@ const SORT_DESCENDING = {
 
 export default function users(state = initialState, action) {
   switch (action.type) {
-    case RECEIVE_USERS: return {
+    case RECEIVE_USERS: {
+      // Get the sorting key from if available from Redux store or else use last_name
+      let sort_key = state.sort.key || 'last_name';
```

Comment:
where is this being used? If `state.sort.key` is null, we set `sort_key` to `username` [here](https://github.com/Abishekcs/WikiEduDashboard/blob/FixSortingStudents/app/assets/javascripts/reducers/users.js#L41) What's the use of setting it to `last_name` here?